stdlib/event/time.lua

```diff
@@ -36,44 +39,54 @@ Event.Time.daily = script.generate_event_name()
 Event.register(defines.events.on_tick,
     function()
         for idx, surface in pairs(game.surfaces) do
-            local day_time = math.fmod(Event.Time.get_day_time(idx), 1)
-            local day_time_minutes = math.floor(day_time * 24 * 60)
+            if not surface.freeze_daytime then
+                local day_time = math.fmod(get_day_time(idx), 1)
+                local day_time_minutes = math.floor(day_time * 24 * 60)
```

Comment:
Well their best practice doesn't even work here because Lua has no 'continue' operator. You could use a goto to skip the loop iteration with the guard clause, but I'm not convinced it's a good idea. I did notice why it was indented and wasn't complaining about it specifically, after all. ;)